newIDE/app/src/SceneEditor/Toolbar.js

```diff
@@ -42,10 +45,45 @@ export class Toolbar extends PureComponent<Props> {
     return (
       <ToolbarGroup lastChild>
         {this.props.showPreviewButton && (
-          <ToolbarIcon
-            onClick={this.props.onPreview}
-            src="res/ribbon_default/preview32.png"
-            tooltip={t`Launch a preview of the scene`}
+          <ElementWithMenu
```

Comment:
We have here twice the same elements (this one, and the next one: showNetworkPreviewButton).

I think it's a good time to make a single component for these. (Taking as props all the function and booleans required: previewFirstSceneName, setScenePreview, etc...).

This will be super useful to:
- Avoid this code duplication, so easier to maintain and we're 100% sure the buttons are always consistent (imagining changing one and not the other... not very professional looking).
- Later refactor the toolbar if we want to show these buttons in the middle of the toolbar.

Can you have a look at making a new component file for these? Using <React.Fragment> to return two buttons without a parent div :)